<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>MiniC Source Code</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTextEdit" name="sourceTextEdit">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="changeStyleButton">
      <property name="text">
       <string>Change Style</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="lexical">
       <attribute name="title">
        <string>lexical</string>
       </attribute>
       <widget class="QTextEdit" name="lexicalTextEdit">
        <property name="geometry">
         <rect>
          <x>-10</x>
          <y>0</y>
          <width>411</width>
          <height>461</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="parser">
       <attribute name="title">
        <string>parser</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="openFile"/>
   <addaction name="saveFile"/>
   <addaction name="lexicalFile"/>
   <addaction name="parseFile"/>
   <addaction name="assemblyFile"/>
   <addaction name="clearFile"/>
  </widget>
  <action name="lexicalFile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>lexical</string>
   </property>
   <property name="toolTip">
    <string>lexical the code in textEdit</string>
   </property>
  </action>
  <action name="openFile">
   <property name="text">
    <string>open</string>
   </property>
   <property name="toolTip">
    <string>open a exited file</string>
   </property>
  </action>
  <action name="saveFile">
   <property name="text">
    <string>save</string>
   </property>
   <property name="toolTip">
    <string>save a file to local</string>
   </property>
  </action>
  <action name="parseFile">
   <property name="text">
    <string>parse</string>
   </property>
   <property name="toolTip">
    <string>parse the code in textEdit</string>
   </property>
  </action>
  <action name="assemblyFile">
   <property name="text">
    <string>assembly</string>
   </property>
   <property name="toolTip">
    <string>create the assemble instruction</string>
   </property>
  </action>
  <action name="clearFile">
   <property name="text">
    <string>clear</string>
   </property>
   <property name="toolTip">
    <string>clear the textEdit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
